src/components/base/BaseButton.vue

```diff
@@ -8,14 +8,12 @@ const props = defineProps({
   },
   buttonSize: {
     type: String,
-    default: "reg",
+    default: "regular",
```

Comment:
With typescript you could also specify allowed values. https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

buttonSize: 'regular' | 'small' | etc

Reply:
I was planning to do it as some point down the line - my practical TS experience is limited but I have seen similar approaches in other ppl code